tests/models/test_user.py

```diff
@@ -420,16 +420,22 @@ def test_merge(self, mock_get_mailchimp_api):
             ],
         }
 
+        # Fields in which the two users may differ
+        different_fields = [
+            'date_modified'
+        ]
+
         # from the explicit rules above, compile expected field/value pairs
         expected = {}
         expected.update(calculated_fields)
         for key in default_to_master_user_fields:
             expected[key] = getattr(self.user, key)
 
         # ensure all fields of the user object have an explicit expectation
+        # except for 'different_fields'
         assert_equal(
             set(expected.keys()),
-            set(self.user._fields),
+            set(self.user._fields).difference(different_fields),
```

Comment:
This isn't bad, but is there a way you could use the calculated_fields structure to get the proper expected value rather than skipping it entirely? Even if it's not much better information, I'd rather use the existing structure to do the test than to add another exception. If it's too crazy, then it'll probably be fine this way.